styles.css

```diff
@@ -7,12 +7,18 @@
 body {
   margin: 0;
   padding: 0;
+  font-family: 'Source Sans Pro', sans-serif;
```

Comment:
I'm not sure why we're not getting a merge conflict here, but since you already added this line in another PR you can remove it from this one.

Reply:
Ok thank you for the feed back! I'll fix it.